README.md

```diff
@@ -1,3 +1,347 @@
-# Q.U.A.I.L. QUAIL Univeral Analytics of Informatic Linkages
+# Q.U.A.I.L. QUAIL Universal Analytics of Informatic Linkages
 This is a project used to pull down and analyze data from REDCap. It places the project
 metadata and data into sqlite databases arranged by form.
+
+![Python Version](https://img.shields.io/pypi/pyversions/Django.svg)
+![Build Status](https://img.shields.io/codeship/d6c1ddd0-16a3-0132-5f85-2e35c05e22b1.svg)
+
+```
+
+
+
+   ____  _    _         _____ _      
+  / __ \| |  | |  /\   |_   _| |     
+ | |  | | |  | | /  \    | | | |     
+ | |  | | |  | |/ /\ \   | | | |     
+ | |__| | |__| / ____ \ _| |_| |____
+  \___\_\\____/_/    \_\_____|______|
+
+   Q.U.A.I.L. Universal Analytics of Informatic Linkages
+
+
+```
+
+# Installation
+
+ 1. Run ` $ python setup.py install `
+ 2. Check the install ` $ quail`,
+    you should get some output that looks something like this.
+
+    ```$ quail
+    Usage: quail install <root>
+    quail redcap generate ( <quail.conf.yaml> <project_name> <token> <url> ) [-i --initialize]
+    quail redcap get_meta (<project_name>) [ -q <quail.conf.yaml> ]
+    quail redcap get_data (<project_name>) [ -q <quail.conf.yaml> ]
+    quail redcap gen_meta (<project_name>) [ -q <quail.conf.yaml> ]
+    quail redcap gen_data (<project_name>) [ -q <quail.conf.yaml> ]
+    quail redcap make_import_files (<project_name>) [ -q <quail.conf.yaml> ]
+
+
+    ```
+  3. Goto ` $ cd examples/hcvprod_pull/DANGER ` folder. You may have to make
```

Comment:
Maybe we should suggest that nothing should go in the code repository. That dodges a lot of the issues that could happen with PHI

Reply:
caveat emptor:

I wrote: "This is up TO You to make sure you don't commit live data to github." in step 4